.buildkite/pipeline.yml

```diff
@@ -49,3 +49,24 @@ steps:
           run: goreleaser
           shell: false
           tty: true
+
+  - block:
+      label: ":package: Want to create a release from this commit?"
+      branches:
+        - 3.x
+        - main
+      blocked_state: passed
+
+  # this tags the commit with the input from the previous block step and pushes it to github
+  # that will trigger the buildkite-cli-release pipeline off the tag which will create a release in github
+  - label: ":rocket: Pushing a tag to release"
+    command: "echo git tag ..."
```

Comment:
We'll actually want to put this in a script I think. We want to construct the tag value based on block step fields and verify that its newer, etc than any current tags